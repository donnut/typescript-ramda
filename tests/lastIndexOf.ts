import R_lastIndexOf = require('../ramda/dist/src/lastIndexOf');

declare const string: string;
declare const string_array: string[];

// @dts-jest:pass:snap
R_lastIndexOf(string);
// @dts-jest:pass:snap
R_lastIndexOf(string, string);
// @dts-jest:pass:snap
R_lastIndexOf(string, string_array);
