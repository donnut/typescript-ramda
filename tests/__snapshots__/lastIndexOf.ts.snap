// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.lastIndexOf(string) 1`] = `"(list: string[] | ArrayLike<string>) => number"`;

exports[`R.lastIndexOf(string, string) 1`] = `"number"`;

exports[`R.lastIndexOf(string, string_array) 1`] = `"number"`;
